packages/lab/src/stepped-tracker/Step.Connector.tsx

```diff
@@ -0,0 +1,28 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+
+import stepConnectorCSS from "./Step.Connector.css";
+
+export namespace StepConnector {
+  export interface Props {
+    className?: string;
+  }
+}
+
+const withBaseName = makePrefixer("saltStepConnector");
+
+export function StepConnector({ className }: StepConnector.Props) {
+  const targetWindow = useWindow();
+
+  useComponentCssInjection({
+    testId: "salt-step-connector",
+    css: stepConnectorCSS,
+    window: targetWindow,
+  });
+
+  return <div aria-hidden className={clsx(withBaseName(), className)} />;
```

Comment:
Not entirely sure why this needs to be a standalone component when only aria attribute and classname is here

Reply:
It's due to the css applied to it, trying to make files smaller, thus more digestible and comprehendible